### ISSUE TITLE
map port

### ISSUE THREAD

**Issue by mindbl4ster** [TaskForce47/Domination-Lingor#48]
- [ ] Chernarus
- [ ] Chernarus Winter
- [ ] Sahrani
- [ ] Isla Duala
- [ ] Reshmaan?

**Comment by christian2526** [comment]
Was haltet ihr davon, wenn nach jeder gewonnenen Mission eine andere kommt? (Neue Map, neuer Modus nicht zwingend)

**Comment by mindbl4ster** [comment]
gut?

**Comment by mindbl4ster** [comment]
war ja auch der plan ... aber dafür brauchen wa halt die ports

**Comment by christian2526** [comment]
Natürlich. Also ich mache die Tage noch den Vehicle Service Point komplett und fange dann an auf Isla Duala zu Porten. Die Map ist perfekt für Domination.

**Comment by mindbl4ster** [comment]
wie wärs mit chernarus? oder ganz altmodisch tanoa/altis? :D

**Comment by christian2526** [comment]
Cherno finde ich persönlich langweilig, da kennt jeder jeden mini stein. Tanoa/Altis kommt irgendwann, sind aber nicht so schön meiner Meinung nach.
